src/vinylcontrol/vinylcontrolprocessor.cpp

```diff
@@ -82,7 +82,7 @@ void VinylControlProcessor::run() {
     QThread::currentThread()->setObjectName(QString("VinylControlProcessor %1").arg(++id));
 
     while (!m_bQuit) {
-        Event::start("VinylControlProcessor");
+        //Event::start("VinylControlProcessor");
```

Comment:
This will not compile now after removing `//`. Please delete all those unused events.